packages/vats/src/core/boot-psm.js

```diff
@@ -218,6 +219,9 @@ export const buildRootObject = (vatPowers, vatParameters) => {
       assert.typeof(name, 'string');
       return consume[name];
     },
+    // ??? any more dangerous than produceItem/consumeItem?
```

Comment:
@warner WDYT of using `getPromiseSpace()` instead of `produceItem` and `consumeItem`? Okay if I remove the latter?